bootstrapping_decision_making.md

```diff
@@ -1,34 +1,34 @@
-# Bootstrapping Decision-Making Bodies
+# Bootstrapping Subproject Councils
 
-As the new Jupyter governance model is approved and implemented, we are moving from an informal Subproject governance model to a formal model. As part of this process, each Jupyter Subproject will need to establish a formal decision-making body, including deciding its size and who its members are. There are a number of challenges associated with this:
+As the new Jupyter governance model is approved and implemented, we are moving from an informal Subproject governance model to a formal model. As part of this process, each Jupyter Subproject will need to establish a formal Subproject council as its decision-making body, including deciding its size and who its members are. There are a number of challenges associated with this:
```

Comment:
We have been trying to be consistent in capitalizing the formal nouns in the governance model. Maybe council should be capitalized to follow the pattern?

Reply:
I audited capitalization here and other places in a new commit I just pushed.